### ISSUE TITLE
`version` field is confusing

### ISSUE THREAD

**Issue by sellout** [DeterminateSystems/flake-schemas#16]
Multiple times now I have forgotten that the `version` field in a schema refers to the version of the `schemas` schema, and is not a version number for the schema that is being defined.

That sentence is confusing, too, so let me try again. When I modify `schemas.projectConfigurations` I am tempted to increment `schemas.projectConfigurations.version`, but that is not the right thing to do.

I don't have a great suggestion, but `schemasVersion` might be an improvement (note that it's plural, as `schemaVersion` doesn't seem any less confusing than `version`).

**Comment by Malix-Labs** [comment]
Indeed
